services/QuillLMS/engines/evidence/app/models/evidence/activity.rb

```diff
@@ -121,6 +123,17 @@ def url
       "evidence/#/activities/#{id}/settings"
     end
 
+    def evidence_research_gen_ai_activity
+      evidence_research_gen_ai_activities&.first
+    end
```

Comment:
Purely stylistic, so  take it or leave it, but this could also be a one-line method:
```ruby
def evidence_research_gen_ai_activity = evidence_research_gen_ai_activities&.first
```